pulseaudio.if

```diff
@@ -293,6 +297,43 @@ interface(`pulseaudio_manage_home_files',`
 	pulseaudio_filetrans_home_content($1)
 ')
 
+########################################
+## <summary>
+##      Manage pulseaudio session tmp dirs.
+## </summary>
+## <param name="domain">
+##      <summary>
+##      Domain allowed access.
+##      </summary>
+## </param>
+#
+interface(`pulseaudio_manage_session_tmp_dirs',`
+        gen_require(`
+                type pulseaudio_home_t;
+        ')
+
+        manage_dirs_pattern($1, pulseaudio_home_t, pulseaudio_home_t)
+')
+
+########################################
+## <summary>
+##      Manage pulseaudio session tmp files.
+## </summary>
+## <param name="domain">
+##      <summary>
+##      Domain allowed access.
+##      </summary>
+## </param>
+#
+interface(`pulseaudio_manage_session_tmp_files',`
+        gen_require(`
+                type pulseaudio_home_t;
+        ')
+
+        manage_files_pattern($1, pulseaudio_home_t, pulseaudio_home_t)
+        allow $1 pulseaudio_home_t:sock_file { create unlink };
```

Comment:
Are these permissions sufficient? Domains like systemd_logind_t have some generic permissions which may not be true for other domains.